packages/mui-base/src/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import * as Tooltip from '@base_ui/react/Tooltip';
+import { createRenderer, screen } from '@mui/internal-test-utils';
+import { expect } from 'chai';
+
+describe('<Tooltip />', () => {
```

Comment:
On the tests, I think it would be great to have more tests (especially overtime). I could see most of Material UI Tooltip test in Base UI. It will help reduce the likeliness of introducing regressions from Material UI v7 perspective.

I saw we mostly kept https://github.com/mui/material-ui/pull/18458.